components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -82,6 +85,7 @@ @implementation MDCTabBarView
 - (instancetype)init {
   self = [super init];
   if (self) {
+    _defaultRippleColor = [[UIColor alloc] initWithWhite:0 alpha:(CGFloat)0.16];
```

Comment:
_Tip:_ I believe this is supposed to be 0.12 not 0.16.

Reply:
Oh, it's not matching anything from our design guidelines. I can make it `0.12` if you like

Reply:
FYI: This came from MDCRippleView's implementation.

Reply:
I'm fairly certain its supposed to be `0.12` I'll confirm with @yarneo today.

Reply:
Ok. I'm going to leave it for now since it matches Ripple's implementation. If it needs to change, then I suspect many components will have to be updated.